src/components/InventoryTabs/ConventionalSystems/ConventionalSystemsTab.test.js

```diff
@@ -125,13 +119,19 @@ describe('ConventionalSystemsTab', () => {
     systemProfileStore: { systemProfile: { loaded: false } },
   };
 
-  const mount = (children, store) =>
-    enzymeMount(
-      <ReactRouterDOM.MemoryRouter>
+  const render = (children, store) =>
+    _render(
```

Comment:
Usually when prepending `_` it is meant to signal that this const/variable is not actually used.

Maybe the function could be named `renderWithProviders` to would allow avoiding the renaming of the `render` import?